tests/acceptance/features/webUIFiles/fileDetails.feature

```diff
@@ -115,6 +115,22 @@ Feature: User can open the details panel for any file or folder
     When the user switches to "links" tab in details panel using the webUI
     Then the "links" details panel should be visible
 
+  @issue-2150
+  Scenario: without any share the shared-with-others page should be empty
+    When the user browses to the shared-with-others page using the webUI
+    Then folder "simple-folder" should be listed on the webUI
+    And file "data.zip" should be listed on the webUI
+    #Then folder "simple-folder" should not be listed on the webUI
```

Comment:
Do we have any steps to check if the list is empty on the webUI?

Reply:
done using `Then there should be no files/folders listed on the webUI`